lib-ee/emqx_ee_bridge/src/kafka/emqx_bridge_impl_kafka_producer.erl

```diff
@@ -143,7 +143,7 @@ on_query(_InstId, {send_message, Message}, #{message_template := Template, produ
     %% If the producer process is down when sending, this function would
     %% raise an error exception which is to be caught by the caller of this callback
     {_Partition, _Pid} = wolff:send(Producers, [KafkaMessage], {fun ?MODULE:on_kafka_ack/3, [#{}]}),
-    {async_return, ok}.
```

Comment:
This should be kept.  Otherwise it'll always increase the success counter.  Wolff already handles that counter (and the failure).

Reply:
then it becomes a failure counter.
after this fix, it double counts success. 
fixed it in my force push.